### PR TITLE
Re-add internal 7-day signal smoothing -- v2

### PR DESCRIPTION
**Pull request by capolitsch**

### CODE REVIEW

File: forecasters/aardvark/R/modeling.R [hunk 1]
```diff
@@ -317,12 +291,49 @@ make_cv_glmnet <- function(alpha = 1, fdev = 0, mnlam = 100, n_folds = 10){
...
     return(preds)
   }
 }
+
+#' @importFrom covidcast aggregate_signals
+#' @import evalcast
+long_to_wide <- function(df){
+  # Manipulate evalcast df to the wide format previously used during evalforecast era
+  # This is a really hacky way to circumvent the issue while GitHub issue #269 is pending
+  if ( nrow(unique(df %>% select(data_source, signal, geo_value, time_value))) < nrow(df) ){
```
Comment (dajmcdon) [inline forecasters/aardvark/R/modeling.R:307]: Do you still need this? `evalcast` can return a signals list or in wide format. I see that 269 is still open.

File: forecasters/aardvark/R/modeling.R [hunk 2]
```diff
@@ -317,12 +291,49 @@ make_cv_glmnet <- function(alpha = 1, fdev = 0, mnlam = 100, n_folds = 10){
...
+  names(df2)[which(substr(names(df2),1,5) == "value")] <- "value"
+  df2$variable_name <- "jhu-csse-confirmed_incidence_num"
+  df <- bind_rows(df1, df2)
+  match.string.2 <- with(df, paste0(variable_name, geo_value, time_value))
+  df$issue <- df.tmp$issue[match(match.string.2, match.string.1)]
+  df <- df %>% mutate(location = evalcast:::abbr_2_fips(df$geo_value)) %>%
+    select(location, geo_value, variable_name, value, time_value, issue)
+  df$value <- as.double(df$value)
+  return(df)
+}
```
Comment (dajmcdon) [inline forecasters/aardvark/R/modeling.R:339]: Each file is supposed to end with an empty line.

File: forecasters/aardvark/R/signal_smoothing.R [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        group_by(location) %>%
+        arrange(time_value) %>%
+        mutate(smoothed_value = rollmean(value, h, align = "right", fill = "extend")) %>%
+        ungroup
+      return(smoothed_dat)
+    }
+  }
+  
+  return(kernel_smoother)
+}
```
Comment (dajmcdon) [inline forecasters/aardvark/R/signal_smoothing.R:34]: Same here.

**Review by dajmcdon — APPROVED** [review]

**Comment by dajmcdon** [comment]
@capolitsch I've approved since changes are minor. You should be able to merge on your own when ready. But note the conflicts as well.

**Comment by capolitsch** [comment]
> @capolitsch I've approved since changes are minor. You should be able to merge on your own when ready. But note the conflicts as well.

Thanks, Dan. This is the first time I've experienced the conflicts warning in a pull request. What exactly is the situation here? They just seem to point out differences between the branches, and they are all differences in which I want the dev-aardvark version merged to the develop branch.

**Comment by dajmcdon** [comment]
I just fixed the conflicts.

**Review by dajmcdon — APPROVED** [review]

**Pull request merged**